src/App.vue

```diff
@@ -45,6 +37,14 @@ export default {
     font-family: 'Roboto Condensed', sans-serif;
   }
 
+  #app {
```

Comment:
Whats the plan for styling the apps vh when content extends 100vh. 

min-vh might be better option here

Reply:
We have a card on the project board to decide on this when we have actual data to work with and can try out different options. We've discussed an infinite scroll situation or pagination, but we're open to other ideas!

Reply:
But yes, good call on min-vh being an option!

Reply:
Coolio, merging